python/pyspark/pandas/tests/test_series.py

```diff
@@ -3020,6 +3020,67 @@ def test_eq(self):
         with self.assertRaisesRegex(ValueError, "Lengths must be equal"):
             psser == other
 
+    def test_combine(self):
+        pdf = pd.DataFrame(
+            {"s1": [330.0, 160.0, np.nan], "s2": [345.0, 0.0, 30.0], "s3": [345.0, 0.0, 30.0]}
+        )
+        psdf = ps.from_pandas(pdf)
+
+        self.assert_eq(
+            pdf["s1"].combine(pdf["s2"], max),
+            psdf["s1"].combine(psdf["s2"], max),
+        )
+        self.assert_eq(
+            pdf["s1"].combine(pdf["s2"], max, fill_value=100),
```

Comment:
Question: Does fill_value work in pandas actually? Looks like we need to find a real case to fill the value I guess.
```
>>> pdf["s1"].combine(pdf["s2"], max, fill_value=100)
0    345.0
1    160.0
2      NaN
dtype: float64
```